lckernel/cad/const.h

```diff
@@ -4,6 +4,7 @@
 
 #define LCTOLERANCE 1.0e-10
 #define LCARCTOLERANCE 1.0e-10
+#define BBHEURISTIC 1.0e-5
```

Comment:
What does BBHEURISTIC exactly do/tells us? I see it's used during hight tests, but why not use LCTOLERANCE?